### PR TITLE
fix: deserialize IP address if returned by API

### PR DESCRIPTION
**Pull request by MrPrashantT**
PR for issue [28773](https://app.shortcut.com/passbase-inc/story/28773/node-sdk-does-not-deserialize-expose-ip-address-ramp).

### CODE REVIEW

**Review by mariusvirb — CHANGES_REQUESTED** [review]
Please remove `package-lock.json`, otherwise lg2m

**Comment by MrPrashantT** [comment]
> Please remove `package-lock.json`, otherwise lg2m

Done @mariusvirb

File: src/models/Identity.ts [hunk 1]
```diff
@@ -21,8 +23,9 @@ export class Identity {
   owner: IdentityOwner | null;
   score: number;
   resources: IdentityResource[];
-  metadata: {[x: string]: any} | null;
+  metadata: { [x: string]: any } | null;
   watchlist: Watchlist | null;
+  network: {} | null;
```
Comment (mariusvirb) [inline src/models/Identity.ts:28]: Just noticed, this should be `Network`.

**Review by mariusvirb — CHANGES_REQUESTED** [review]
@MrPrashantT, this does not seem right :thinking: it seems package-lock **does** existing in our main branch :exploding_head: Well, since we didn't update any dependencies, please revert this file to it's original form.

File: src/models/Identity.ts (hunk 1 above)
Comment (MrPrashantT) [reply src/models/Identity.ts]: Both fixed in latest commit.

**Review by mariusvirb — DISMISSED** [review]
Perfect :lipstick:

**Review by mariusvirb — APPROVED** [review]

**Pull request merged**